README.rst

```diff
@@ -75,19 +75,18 @@ Running the Tests
 -----------------
 
 To run the tests you will need to install
-`lettuce <http://packages.python.org/lettuce/tutorial/simple.html>`_
+`nose <https://nose.readthedocs.org/en/latest/>`_
 
 .. code-block:: bash
 
-    $ pip install lettuce
+    $ pip install nose
```

Comment:
since we have `nose` in the `setup_requires` list, i think this won't be necessary: it's enough to call `python setup.py develop` (which is needed anyway).